develop/fpm-alpine/Dockerfile

```diff
@@ -0,0 +1,86 @@
+FROM php:7.1-fpm-alpine
+LABEL maintainer="Philipp Holzer <admin@philipp.info>"
+
+ENV IMAGICK_PECL 3.4.3
+ENV AUTOINSTALL false
+
+# entrypoint.sh and cron.sh dependencies
+RUN set -ex; \
+    \
+    apk add --no-cache \
+        rsync \
+        git \
+    ; \
+    \
+    rm /var/spool/cron/crontabs/root; \
+    echo '*/10 * * * * cd /var/www/html && php -f bin/worker.php' > /var/spool/cron/crontabs/www-data
+
+# install the PHP extensions we need
+# see https://friendi.ca/resources/requirements/
+RUN set -ex; \
+    \
+    apk add -U --no-cache --virtual .build-deps \
+        libxml2-dev \
```

Comment:
I don't see an email system in the list, how do the users get their notification emails?

Reply:
How is Friendica normally using an email system? I know from the `dev` resources of the Friendica repository that you automatically install postfix and use it as a mail-server.

But how is it in production? For my environment, I do have a SMTP server for my local network where I'm routing the mails through.

One guess is that I'm automatically install `sendmail` and add "optional" SMTP-environment variables for the docker instances. What do you think?